v0/destinations/appcues/data/AppcuesPageProfile.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
are we following this kind of value mapping in json for this destination? @arnab-p